be/tool/beck.c

```diff
@@ -427,6 +430,12 @@ static bool  dry_run = false;
 	M0_LOG(level, prefix " %li %s %hu:%hu:%u", s->s_off, recname(r), \
 	       (tag)->ot_version, (tag)->ot_type, (tag)->ot_size)
 
+static void  sig_handler(int num)
```

Comment:
remove extra space between "void" and "sig_handler".

Reply:
done.